packages/botonic-cli/src/util/file-system.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  existsSync,
+  mkdirSync,
+  mkdtempSync,
+  readdirSync,
+  readFileSync,
+  rmdirSync,
+  writeFileSync,
+} from 'fs'
+import { copySync } from 'fs-extra'
+
+export function pathExists(path: string): boolean {
+  return existsSync(path)
+}
+
+export function readDir(path: string): string[] {
+  return readdirSync(path)
+}
+
+export function readJSON(path: string): any {
+  const fileContent = readFileSync(path, 'utf8')
+  if (!fileContent) return undefined
```

Comment:
this check is necessary in order to not doing JSON.parse('') of an empty content, o.w. it will cause an error which will break the entire developer flow.